### ISSUE TITLE
[Adaptive Card iOS - Input.Number.Json]: After entering the incorrect number, no error message is getting displayed. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8429]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.Number' and activate it.
7. Navigate to the 'Enter a Number' text edit field
8. Enter incorrect number and perform right swipe.
9. Double tap on OK button and observe the issue.

**Actual Result:** 
On entering the incorrect number under the 'Enter a number' text edit field, there is no error message is getting displayed. Currently outer boundary of edit field is getting highlighted with Red color.

**Expected Result:**
An error message should be displayed when user enters the incorrect number and voice over should convey the same.

**User Impact:**
It will impact the screen reader user as they will not able to know what will be the correct value also they will not able to know what is the main error they are getting.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228807067-194386b2-b4a4-4a37-9c4b-8f640bbf0ff4.mov

### Card JSON

```JSON
NA
```
